### ISSUE TITLE
Balance is not updated immediately after Deposit

### ISSUE THREAD

**Issue by irinelT** [tempus-finance/tempus-app#1074]
_This is not always reproducible so its low priority_

### Prerequisite

- Connect MetaMask to Tempus App ([details](https://docs.tempus.finance/user-guides/how-to-connect-a-wallet))
- Switch MetaMask to Fantom network ([details](https://docs.fantom.foundation/tutorials/set-up-metamask))

### Steps to reproduce

1. Navigate to https://tempus-app-testnet-v2.web.app/
2. **Expand WETH. (not applicable for other assets)**
3. Manage.
4. Deposit to Fixed Yield.
5. Select WETH in "From" dropdown
6. Enter nonzero amount.
7. Approve.
8. Execute.
9. Wait for "Deposit Successful" confirmation.

### Expected behaviour

- Balance should be updated immediately

### Actual behaviour and a screenshot
- Balance is updated only after refresh.

![image](https://user-images.githubusercontent.com/98878781/155292011-e0694a47-19a5-4d27-af14-6a458081b5ee.png)

**Comment by lazarmitic** [comment]
Code here seems fine, we are manually triggering balance update after execute and updating the UI. Since this is not always happening I would assume it's a problem with RPC endpoint that we can't control. Not sure there is something we can do here on the app side.

**Comment by lazarmitic** [comment]
We can add some kind of retry logic in case balance update request fails.

**Comment by lazarmitic** [comment]
But I would not do this for 2.1, maybe later on when we polish other more urgent things.

**Comment by ShawTim** [comment]
https://github.com/tempus-finance/tempus-app/issues/1118
we can test this issue once we have #1118 resolved and with a bad fantom network

**Comment by irinelT** [comment]
Predecesor #1118 is not yet fixed, so moving this out of release.

**Comment by irinelT** [comment]
Out of scope.
